packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py

```diff
@@ -340,17 +342,18 @@ def expected_service_running(
             with websocket.expect_event("framereceived", waiter, timeout=timeout):
                 if press_start_button:
                     _trigger_service_start(page, node_id)
-        except TimeoutError:
-            if waiter.is_progress_succesfully_finished() is False:
+
+                yield service_running
+
+        except PlaywrightTimeoutError:
+            if waiter.get_last_call_return() is False:
```

Comment:
in this case this is enough
```suggestion
            if not waiter.get_last_call_return():
```